### PR TITLE
Goddess' Ancient Garden / Escanciu Village / Fedimian Suburbs

### PR DESCRIPTION
**Pull request by terrapin47**
Changes:
Neryshkus Sanctum Demolition -> Neryshkus Sanctum Ruins
@imcgames I'm suggesting this be an update to the wiki as well, it should be 'ruins' and not 'demolition'

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -1956,22 +1956,22 @@ QUEST_LV_0100_20150317_001951	{memo X}The Bishop told me to send the Revelator t
...
 QUEST_LV_0100_20150317_001960	$When you use the stone on Infroburks and defeat them, they will automatically come to me.{nl}Okay then, I am counting on you.
-QUEST_LV_0100_20150317_001961	She is the most impressive archer in history.{nl}She defeated Ruklys, who was a traitor and the great swordsman with just an arrow.{nl}
-QUEST_LV_0100_20150317_001962	That's why all archers want to purchase things that are related to her even if they are expensive.{nl}She is like the Goddess to us archers.
-QUEST_LV_0100_20150317_001963	$There was a priest called Maven in the past.{nl}He is the person who built the Great Cathedral.{nl}
+QUEST_LV_0100_20150317_001961	She is the most impressive archer in history.{nl}She defeated Ruklys, who was a traitor and a great swordsman, with a single arrow.{nl}
+QUEST_LV_0100_20150317_001962	That's why all archers want to purchase things that are related to her, even if they are expensive.{nl}She is like a Goddess to us archers.
+QUEST_LV_0100_20150317_001963	$There was a priest called Maven in the past.{nl}He is the person who built the great cathedral.{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Why was this changed?

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -6108,10 +6108,10 @@ QUEST_LV_0100_20150317_006103	Check the Military Backpack
...
 QUEST_LV_0100_20150317_006105	{memo X}
 QUEST_LV_0100_20150317_006106	$Commence Operation Wooden Village
-QUEST_LV_0100_20150317_006107	$Found the plan for Operation Wooden Village in the backpack. The plan was to defeat the Zolems around Neryshkus Sanctum Demolition, then using Zolem Magic Stones to summon Unicorn, and then defeating it. The Wandering Spirit seems to have been in an accident while carrying out the plan. Continue the plan to comfort the soul.
+QUEST_LV_0100_20150317_006107	$Found the plan for Operation Wooden Village in the backpack. The plan was to defeat the Zolems around Neryshkus Sanctum Ruins, then using Zolem Magic Stones to summon Unicorn, and then defeating it. The Wandering Spirit seems to have been in an accident while carrying out the plan. Continue the plan to comfort the spirit.
 QUEST_LV_0100_20150317_006108	$Prepare to lure Unicorn
 QUEST_LV_0100_20150317_006109	$Collected enough Zolem Stones. Put the stone in the abandoned bag written in the command and lure the unicorn.
-QUEST_LV_0100_20150317_006110	$Defeat Unicorn and acquire %s
+QUEST_LV_0100_20150317_006110	$Acquire %s
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Should be: Collect &s by defeating Zolem/Zolems

File: QUEST_LV_0100.tsv (hunk 1 above)
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: Mistake, I'll change it back.

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by terrapin47** [comment]
Resolved conflicts.

**Comment by imcgames** [comment]
Updated Wiki :D

**Comment by terrapin47** [comment]
Resolved conflicts, and thanks!

**Comment by imcgames** [comment]
:+1:
